forte/data/ontology/top.py

```diff
@@ -515,7 +514,7 @@ class MultiPackGroup(MultiEntry, BaseGroup[Entry]):
     of members.
     """
     member_type: str
-    members: Optional[FList[Entry]]
+    members: List[Tuple[int, int]]
```

Comment:
Hmm, I thought we won't change the interface the users see. @mylibrar any idea here?

Reply:
This was actually a small bug that I identified in the previous PR (#882). This keeps the structure of `MultiPackGroup` consistent with its implementation (since it was previously implemented as a list of tuples).

Reply:
I think the interface for `Group` should be that the members are exposed as a list of entries? I think as the interface, a tuple of `int` is a bit weird. When you say "keep the structure consistent with implementation", where is the implementation you are referring to?

Reply:
So for `Group` type entries, we fetch all members of the group using `get_members` and if you see the implementation of [that](https://github.com/Pushkar-Bhuse/forte/blob/23e8ac88092c70fe6696f827a4dec9257bd27683/forte/data/ontology/top.py#L540) you will see that we return a list of entries.

The `members` attribute is a `dataclass` attributes which means that it will be stored in the `DataStore`. Thus it should be in primitive python types. Moreover, this `members` list does the job of [this](https://github.com/asyml/forte/blob/d6b137d315a958475568ebda85fde9c2d98afb54/forte/data/ontology/top.py#L630) members list in the previous implementation which was just used to store the member list in `DataStore` and is thus never exposed to the user directly.

Reply:
One related interface problem is on ontology generation. I think our ontology code generator, will anything happen there.

The reason that I am worried about the interface is that  MultiPackGroup is a user-facing class anyway, so the public member of it could have some implication/impact on users.

I will let @mylibrar decide how to deal with this here.